dist/components/product/_product.dust

```diff
@@ -0,0 +1,38 @@
+{#product}
+<div class="c-product {?isHighlight}c--highlight{/isHighlight}" role="group">
+    <a class="c-product__title" href="{url}" title="{title}" rel="external">
+        <h2>{title}</h2>
+    </a>
+
+    {#image}
+        <div class="c-product__image">
+            <a href="{url}" title="{title}" rel="external">
+                <img src="{path}" alt="{title}" />
+            </a>
+        </div>
+    {/image}
+
+    <span class="c-product__price" aria-label="Price is {price}">{price}</span>
+
+    {#isDiscount}
+        <span class="c-product__price" aria-label="Old price is {price}">{oldPrice}</span>
+
+        <s class="c-product__price c--discount" aria-label="New discounted price is {price}">{discountPrice}</s>
+    {/isDiscount}
+
+    <div class="c-product__controls">
+        {colors} <!-- colors component -->
+        {sizes} <!-- sizes component -->
+        {quantity} <!-- quantity component -->
+        {ratings} <!-- ratings component -->
```

Comment:
I'm not so we should be so prescriptive about what product details are output here, since who knows what kind of information a site has about their products. Many of these might not be applicable, or there may be alternative properties that these simply do not cover.